### ISSUE TITLE
Editing a goal

### ISSUE THREAD

**Issue by jkbdbrnwsk** [x-team/unleash#464]
Clicking on the "three dots" :) should trigger two inputs:

- goal name
- goal description

If text has been already put, user can edit it. Changes applied by hitting enter.

![image](https://cloud.githubusercontent.com/assets/7212882/26694180/de6f17aa-4706-11e7-90ee-d8d75fe89f4b.png)

**Comment by jacekelgda** [comment]
are we thinking about mobile users as well ? Seems mobile UX is being forgotten in here

**Comment by jacekelgda** [comment]
do we want to have `...` meaning `edit` ? 
it usually is a function to show more options

**Comment by jkbdbrnwsk** [comment]
Yeah, as you pointed out in another issue, this can actually trigger a dropdown with a few options such as "Mark as achieved", "Edit", etc.

Speaking of mobile: apart from making the header a bit more mobile friendly, especially on phone viewports (for example by putting each header element below another, not next to each other as it is on desktop), do you have a particular approach in mind? Like this element here: **...** Should it behave somewhat else on mobile, should we actually have a separate, more mobile friendly designs for particular elements, or...? :) Looking for your guidance here. :)
